configs/siklu_80X0_defconfig

```diff
@@ -79,13 +85,3 @@ BR2_PACKAGE_TAR=y
 BR2_TARGET_ROOTFS_SQUASHFS=y
 BR2_TARGET_ROOTFS_SQUASHFS4_LZMA=y
 BR2_TARGET_ROOTFS_TAR_GZIP=y
-BR2_PACKAGE_LIBCURL=y
-BR2_PACKAGE_LIBCURL_CURL=y
-BR2_PACKAGE_LIBCURL_PROXY_SUPPORT=y
-BR2_PACKAGE_LIBCURL_COOKIES_SUPPORT=y
-BR2_PACKAGE_LIBCURL_EXTRA_PROTOCOLS_FEATURES=y
```

Comment:
Why remove BR2_PACKAGE_LIBCURL_EXTRA_PROTOCOLS_FEATURES?
Can you please review all changes in the buildroot config?

Reply:
I didn't remove it, if you rum make menuconfig you will see it 'y'

Reply:
@shmulik-aha can you please check if the `copy sw` CLI command works from an external SFTP server? We use curl for it. As I remember, dali runs an SFTP server.

Reply:
@dima1308 I don't know/understand how to check it

Reply:
to check it:
1. Load SW with uploaded buildroot to the device
2. Login into CLI
3. Execute command `copy sw <sftp://<SFTP server IP address>/path/to/SW_image>`

If copy SW works, we have curl support.
you can use dali server as an SFTP server.

Reply:
@dima1308 I don't understand what you wrote.
For download from dali ftp we write:
EH-8010FX-L-D>copy sw ftp://192.168.42.30/shmulika/nxp/siklu-uimage-nxp-enc-10.7.1-18976-2cd185b0a5-dirty

For download from dali sftp I get an error not related to my version:
EH-8010FX-L-D>copy sw sftp://192.168.42.30/shmulika/nxp/siklu-uimage-nxp-enc-10.7.1-18976-2cd185b0a5-dirty

fail to download sftp://192.168.42.30/shmulika/nxp/siklu-uimage-nxp-enc-10.7.1-18976-2cd185b0a5-dirty: Authentication failure
E

Reply:
> @dima1308 I don't understand what you wrote. For download from dali ftp we write: EH-8010FX-L-D>copy sw ftp://192.168.42.30/shmulika/nxp/siklu-uimage-nxp-enc-10.7.1-18976-2cd185b0a5-dirty
> 
> For download from dali sftp I get an error not related to my version: EH-8010FX-L-D>copy sw sftp://192.168.42.30/shmulika/nxp/siklu-uimage-nxp-enc-10.7.1-18976-2cd185b0a5-dirty
> 
> fail to download sftp://192.168.42.30/shmulika/nxp/siklu-uimage-nxp-enc-10.7.1-18976-2cd185b0a5-dirty: Authentication failure E

@shmulik-aha needs to encode your credentials into the URL:
`sftp://username:password@192.168.42.30/shmulika/nxp/siklu-uimage-nxp-enc-10.7.1-18976-2cd185b0a5-dirty`

Reply:
@dima1308 still not working with latest master version (with old buildroot):
EH-8012FX_L>copy sw sftp://shmulika:q123456Q@192.168.1.30/shmulika/nxp/siklu-uimage-nxp-enc-10.7.1-18976-2cd185b0a5
-dirty

sftp://shmulika:q123456Q@192.168.1.30/shmulika/nxp/siklu-uimage-nxp-enc-10.7.1-18976-2cd185b0a5-dirty -> /tmp/sw.imsftp://shmulika:q123456Q@192.168.1.30/shmulika/nxp/siklu-uimage-nxp-enc-10.7.1-18976-2cd185b0a5-dirty -> /tmp/sw.imsftp://shmulika:q123456Q@192.168.1.30/shmulika/nxp/siklu-uimage-nxp-enc-10.7.1-18976-2cd185b0a5-dirty -> /tmp/sw.imsftp://shmulika:q123456Q@192.168.1.30/shmulika/nxp/siklu-uimage-nxp-enc-10.7.1-18976-2cd185b0a5-dirty -> /tmp/sw.imsftp://shmulika:q123456Q@192.168.1.30/shmulika/nxp/siklu-uimage-nxp-enc-10.7.1-18976-2cd185b0a5-dirty -> /tmp/sw.imsftp://shmulika:q123456Q@192.168.1.30/shmulika/nxp/siklu-uimage-nxp-enc-10.7.1-18976-2cd185b0a5-dirty -> /tmp/sw.imsftp://shmulika:q123456Q@192.168.1.30/shmulika/nxp/siklu-uimage-nxp-enc-10.7.1-18976-2cd185b0a5-dirty -> /tmp/sw.imsftp://shmulika:q123456Q@192.168.1.30/shmulika/nxp/siklu-uimage-nxp-enc-10.7.1-18976-2cd185b0a5-dirty -> /tmp/sw.img download warm upCould not open remote file for reading: No such file or directory

Reply:
@shmulik-aha sorry, my bad. Dali is not accessible from the lab devices.
Need to use another server and/or ask Yuri to help.

Reply:
Verified that it is working